db/migrate/20220505152800_create_post.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePost < ActiveRecord::Migration[6.1]
+  def change
+    create_table :posts do |t|
+      t.text :title
+      t.string :text
```

Comment:
- [ ] The title type should be a string, text type should be a text. Please change it.